hypervisor/src/mshv/mod.rs

```diff
@@ -1814,7 +1814,7 @@ impl vm::Vm for MshvVm {
     fn complete_isolated_import(
         &self,
         snp_id_block: IGVM_VHS_SNP_ID_BLOCK,
-        host_data: &[u8],
+        host_data: &[u8; 32],
```

Comment:
Better to be: `host_data: [u8; 32],`, so that it can be directly used for constructing `hv_partition_complete_isolated_import_data` (no need for `host_data[0..32].try_into().unwrap()` anymore).